arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -0,0 +1,689 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "RedisManagementClient",
+    "description": ".Net client wrapper for the REST API for Azure Redis Cache Management Service",
```

Comment:
This is the spec. It will be used to generate the clients for different languages.
"Describes the REST API for Azure Redis Cache Management Service"